cypress-tests/cypress/e2e/PaymentUtils/Commons.js

```diff
@@ -1456,7 +1454,7 @@ export const connectorDetails = {
       Request: {
         currency: "USD",
         customer_acceptance: null,
-        setup_future_usage: "off_session",
+        setup_future_usage: "on_session",
```

Comment:
Change made because now , if you pass off_session the payments will be considered as mandates , payment methods will not show , these test cases will be taken by @prajjwalkumar17